libhb/hb.c

```diff
@@ -409,13 +416,20 @@ void hb_log_level_set(hb_handle_t *h, int level)
     global_verbosity_level = level;
 }
 
+/*
+ * Enable or disable support for OpenCL detection.
+ */
+void hb_opencl_status_set(hb_handle_t *h, int enable_opencl)
```

Comment:
I looked at old code, we have two naming conventions for this type of function:
- hb_dvd_set_dvdnav()
- hb_hwd_set_enable()

So maybe hb_opencl_set_enable() or hb_hwaccel_set_opencl() ?